spring-cloud-gcp-autoconfigure/src/main/java/com/google/cloud/spring/autoconfigure/pubsub/GcpPubSubAutoConfiguration.java

```diff
@@ -406,7 +406,9 @@ public TopicAdminSettings topicAdminSettings(
           TransportChannelProvider publisherTransportChannelProvider) {
     try {
       return TopicAdminSettings.newBuilder()
-          .setCredentialsProvider(this.finalCredentialsProvider)
+              .setUniverseDomain(gcpPubSubProperties.getUniverseDomain())
+              .setEndpoint(gcpPubSubProperties.getEndpoint())
```

Comment:
Are these setters null-safe?

Design document states "**If specified**, set this value..."

Reply:
KMS example shows null checks.